.github/workflows/CI.yml

```diff
@@ -36,6 +36,24 @@ jobs:
       - run: cargo test
         working-directory: pyo3-polars
 
+      - run: cargo test --all-features
```

Comment:
I rather have something like cargo-hack instead of us manually selecting features.